### ISSUE TITLE
Support for ArrayLiteralExpressions

### ISSUE THREAD

**Issue by avin-kavish** [Hookyns/tst-expression#11]
Looks like it's missing the node type, and guard.

**Comment by avin-kavish** [comment]
Here's quick hack I wrote for that,

```ts
interface ArrayLiteralNode extends ExpressionNode {
  kind: tst.ExpressionKind.ArrayLiteralExpression
  elements: ExpressionNode[]
  multiLine: boolean
}

export function isArrayLiteral(
  node: ExpressionNode,
): node is ArrayLiteralNode {
  return node.kind === tst.ExpressionKind.ArrayLiteralExpression
}
```

**Comment by Hookyns** [comment]
Published `tst-expression-transformer@1.2.2` and `tst-expression@2.1.3`.

BTW: Module changed from UMD to CommonJS, so it works with CommonJS and NodeNext modules.
